specification/dimensions/chargetype.md

```diff
@@ -25,17 +23,17 @@ after-the-fact adjustment (e.g., credits), or taxes.
 
 ## Content Constraints
 
-| Constraint      | Value                                    |
-| :-------------- | :--------------------------------------- |
-| Column required | True                                     |
-| Data type       | String                                   |
-| Allows nulls    | False                                    |
-| Value format    | list-of-values                           |
+| Constraint      | Value          |
+| :-------------- | :------------- |
+| Column required | True           |
+| Data type       | String         |
+| Allows nulls    | False          |
+| Value format    | list-of-values |
 
 Allowed values:
 
 | Value      | Description                                                                                                                                                                   |
-|:-----------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| :--------- | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | Adjustment | Any adjustments that are applied after the original usage or purchase record. Adjustments may be related to multiple charges.                                                 |
```

Comment:
Do out-of-cycle bills, refunds and credits apply as an Adjustment?

Reply:
Refunds and credits on those out-of-cycle bills would be adjustments. Other line items would show as they normally would. We're more focused on the cost details than invoice generation today.

Reply:
Let's discuss in Slack if you think there's a gap here we should account for in 1.0.